lib/telemetry_metrics.ex

```diff
@@ -525,12 +540,13 @@ defmodule Telemetry.Metrics do
     end
   end
 
+  # Don't wrap measurement if no conversion is required.
   @spec maybe_convert_measurement(measurement(), conversion_ratio :: non_neg_integer()) ::
           measurement()
-  defp maybe_convert_measurement(measurement, 1) do
-    # Don't wrap measurement if no conversion is required.
-    measurement
-  end
+  defp maybe_convert_measurement(measurement, 1), do: measurement
+
+  @spec maybe_convert_measurement(measurement(), conversion_ratio :: float()) :: measurement()
+  defp maybe_convert_measurement(measurement, 1.0), do: measurement
```

Comment:
I had to do this in order to not wrap the measurement when both units are the same. Maybe there's another way to do this that I haven't thought about?
I would love to hear some suggestions 😄

Reply:
You can do it like in the time unit and make sure that `1` (instead of 1.0) is returned when both units are the same.